tools/deployment/osqueryctl

```diff
@@ -77,7 +81,8 @@ start() {
 
 stop() {
   if [ $OS = "darwin" ]; then
-    launchctl stop $PLIST_PATH
+    launchctl unload $PLIST_PATH
+    rm $PLIST_PATH
```

Comment:
Also, if you don't remove the plist from `Library/LaunchDaemons`, the launchctl will pick it up and start running it whenever the machine restarts.